akka-docs/src/main/paradox/typed/style-guide.md

```diff
@@ -295,3 +300,151 @@ Scala
 
 Java
 :  @@snip [StyleGuideDocExamples.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StyleGuideDocExamples.java) { #public-private-messages-2 }
+
+@@@ div {.group-java}
+
+### Singleton messages
+
+For messages without parameters the `enum` singleton pattern is recommended:
+
+Java
+:  @@snip [StyleGuideDocExamples.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StyleGuideDocExamples.java) { #message-enum }
+
+In the `ReceiveBuilder` it can be matched in same way as other messages:
+
+Java
+:  @@snip [StyleGuideDocExamples.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StyleGuideDocExamples.java) { #message-enum-match }
+
+@@@
+
+@@@ div {.group-java}
+
+## Lamdas vs. method references
```

Comment:
I'd spell it out for a section title
```suggestion
## Lamdas versus method references
```

Reply:
I thought it wouldn't fit in the left ToC, but it does.
Longest is: Functional versus object-oriented style

Reply:
Hrm, the Toc, ok.